.github/actions/verifySignedCommits/verifySignedCommits.js

```diff
@@ -0,0 +1,24 @@
+const _ = require('underscore');
+const core = require('@actions/core');
+const github = require('@actions/github');
+const GitHubUtils = require('../../libs/GithubUtils');
+
+const PR_NUMBER = Number.parseInt(core.getInput('PR_NUMBER'), 10) || github.context.payload.pull_request.number;
+
+GitHubUtils.octokit.pulls.listCommits({
+    owner: GitHubUtils.GITHUB_OWNER,
+    repo: GitHubUtils.APP_REPO,
+    pull_number: PR_NUMBER,
+})
+    .then(({data}) => {
+        const allCommitsSigned = _.every(data, datum => datum.commit.verification.verified);
+
+        if (!allCommitsSigned) {
+            const unsignedCommits = _.filter(data, datum => !datum.commit.verification.verified);
```

Comment:
NAB: Instead of having a separate check of `allCommitsSigned` you could replace it with this
`  const unsignedCommits = _.filter(data, datum => !datum.commit.verification.verified);`
And then in the check:
`if (!_.isEmpty(unsignedCommits)) {`

Reply:
I agree with @neil-marcellini and think it's NNAB (NOT not a blocker 😅)